selfdrive/car/tests/test_fw_fingerprint.py

```diff
@@ -16,6 +19,14 @@
 VERSIONS = get_interface_attr("FW_VERSIONS", ignore_none=True)
 
 
+class FakeSocket:
```

Comment:
The one in process replay either hangs or throws an exception with the empty list being popped from. We really just want these functions to do nothing so the timeouts in IsoTpParallelQuery are hit